src/pages/ReimbursementAccount/ReimbursementAccountForm.js

```diff
@@ -39,9 +40,15 @@ class ReimbursementAccountForm extends React.Component {
         let error = '';
 
         if (!_.isEmpty(this.props.reimbursementAccount.errorModalMessage)) {
-            error = (
-                <Text style={styles.mutedTextLabel}>{this.props.reimbursementAccount.errorModalMessage}</Text>
-            );
+            if (this.props.reimbursementAccount.isErrorModalMessageHtml) {
+                error = (
+                    <RenderHTML html={`<muted-text>${this.props.reimbursementAccount.errorModalMessage}</muted-text>`} />
```

Comment:
So `<muted-text>` is the same as a div with the "muted-text" attribute?

Reply:
Yes, a `<div>` with the styles from `styles.mutedTextLabel` applied.

Reply:
Ah ok, so it'll come out to `<div style="styles.mutedTextLabel">`, but `currentNode.domNode.name === muted-text`?

Reply:
> Ah ok, so it'll come out to `<div style="styles.mutedTextLabel">`, but `currentNode.domNode.name === muted-text`?

Yep, but I think you cannot do directly `<div style="styles.mutedTextLabel">`